src/main/features/claimant-response/routes/check-and-send.ts

```diff
@@ -60,7 +60,7 @@ function renderView (form: Form<StatementOfTruth>, res: express.Response): void
   const directionsQuestionnaireDraft = res.locals.directionsQuestionnaireDraft
   const claim: Claim = res.locals.claim
   const alreadyPaid: boolean = StatesPaidHelper.isResponseAlreadyPaid(claim)
-  const paymentIntention: PaymentIntention = alreadyPaid ? undefined : getPaymentIntention(draft.document, claim)
+  const paymentIntention: PaymentIntention = alreadyPaid || claim.response.responseType === ResponseType.FULL_DEFENCE ? undefined : getPaymentIntention(draft.document, claim)
```

Comment:
should this change be part of this PR? if its not easy to take it out, just leave it! I can understand.

Reply:
we're basically putting all the mediation stuff in with the DQs so it will be needed

Reply:
ok